app/controllers/candidate_interface/restructured_work_history/break_controller.rb

```diff
@@ -0,0 +1,77 @@
+module CandidateInterface
+  class RestructuredWorkHistory::BreakController < RestructuredWorkHistory::BaseController
+    def new
+      @work_break = if params[:start_date] && params[:end_date]
```

Comment:
this can be done within the form
```
WorkHistoryBreakForm.build_from_date_params(date_params)

private 

def date_params
  {
    start_date:  params[:start_date],
    end_data: params[:end_date]
  }
end
 ```

Then add a method to the form which sets the values